pkg/cmd/pulumi/destroy.go

```diff
@@ -130,7 +131,11 @@ func newDestroyCmd() *cobra.Command {
 			}
 
 			proj, root, err := readProject()
-			if err != nil {
+			if err != nil && errors.Is(err, workspace.ErrProjectNotFound) {
```

Comment:
This isn't right? The error won't directly be an ErrProjectNotFound, because you wrapped it another error

Reply:
I learned that this works! `fmt.Errorf` with a `%w` argument wraps errors and allows you to add helpful error messages around basic errors like `fs.ErrWhatever`.

https://github.com/golang/go/blob/release-branch.go1.17/src/fmt/print.go#L574

It's quite powerful.

Reply:
And Is picks up the inner wrapped error?

Reply:
Yep. https://go.dev/play/p/CCkYSjwVvuR